mac/KeymanEngine4Mac/KeymanEngine4Mac/KME/OnScreenKeyboard/KeyView.m

```diff
@@ -27,28 +28,30 @@ @interface KeyView ()
 @property (nonatomic, assign) BOOL isCharacterKey;
 @property (nonatomic, strong) KeyLabel *label;
 @property (nonatomic, strong) NSTextField *caption;
-@property (nonatomic, strong) NSImageView *bitmapView;
-@property (nonatomic, strong) NSColor *bgColor1;
-@property (nonatomic, strong) NSColor *bgColor2;
+@property (nonatomic, strong) NSColor *bgColorRegularKey;
+@property (nonatomic, strong) NSColor *bgColorSpecialKey;
 @property (nonatomic, strong) NSTimer *keyEventTimer;
 @property (readwrite) NSInteger tag;
 @end
 
 @implementation KeyView
 @synthesize tag;
-@synthesize bgColor1, bgColor2;
+@synthesize bgColorRegularKey, bgColorSpecialKey;
 
 - (id)initWithFrame:(NSRect)frame {
+    os_log_t oskKeyLog = os_log_create("org.sil.keyman", "osk-key");
     self = [super initWithFrame:frame];
     if (self) {
+        os_log_with_type(oskKeyLog, OS_LOG_TYPE_DEBUG, "KeyView initWithFrame: %{public}@, bounds: %{public}@, default clipsToBounds %{public}@", NSStringFromRect(frame), NSStringFromRect(self.bounds), self.clipsToBounds?@"YES":@"NO");
```

Comment:
I take it the new logs are intentionally being left in?

Reply:
Yes, I am planning to include more log statements with the unified logging APIs because attaching the debugger to the input method is not practical. I will do this throughout the app for #10997.

With log statements of type DEBUG, the messages are not persisted and are immediately discarded unless debug messages are enabled (disabled by default) and the log is actually being streamed.